scala-thoughts.md

```diff
@@ -0,0 +1,37 @@
+### What's easy to do in Scala? What's not?
+* I was impressed with the variety of methods on lists! I used map and filter, because those
+were methods that I learned in my CS52 class last semester. 
+One of Scala's greatest strengths is how concise it can be,

```

Comment:
Definitely! I love concise code so much. Less code -> less maintenance in my opinion